ports/pdal/vcpkg.json

```
@@ -1,15 +1,109 @@
 {
   "name": "pdal",
-  "version": "1.7.1",
-  "port-version": 13,
+  "version": "2.3.0",
   "description": "PDAL - Point Data Abstraction Library is a library for manipulating point cloud data.",
+  "homepage": "https://pdal.io/",
+  "supports": "!windows , !staticcrt",
```

```suggestion
  "supports": "!windows | !staticcrt",
```
I think it should be this?

---

It is equivalent. If any change, then `!(windows & staticcrt)`.

---

we should better to use `&` here.